opencti-platform/opencti-front/src/private/components/analyses/reports/ReportEditionOverview.jsx

```diff
@@ -84,23 +84,27 @@ const reportMutationRelationDelete = graphql`
   }
 `;
 
+const REPORT_TYPE = 'Report';
+
 const ReportEditionOverviewComponent = (props) => {
   const { report, enableReferences, context, handleClose } = props;
   const { t_i18n } = useFormatter();
 
-  const basicShape = {
-    name: Yup.string().trim().min(2).required(t_i18n('This field is required')),
-    published: Yup.date()
-      .typeError(t_i18n('The value must be a datetime (yyyy-MM-dd hh:mm (a|p)m)'))
-      .required(t_i18n('This field is required')),
+  const { mandatoryAttributes } = useIsMandatoryAttribute(REPORT_TYPE);
+  const basicShape = yupShapeConditionalRequired({
```

Comment:
With this method, you need to make sure that all the fields in the form are present in the shape just below it. For example, here the `createdBy` field on the form is not in the shape.
=> When you delete the author in the field, you have a different behaviour to the other fields.